hack/godep-save.sh

```diff
@@ -33,35 +35,35 @@ REQUIRED_BINS=(
   "./..."
 )
 
+STAGING_REPOS=(
+  "k8s.io/api"
+  "k8s.io/client-go"
+  "k8s.io/apiserver"
+  "k8s.io/apimachinery"
+  "k8s.io/kube-aggregator"
+  "k8s.io/apiextensions-apiserver"
+  "k8s.io/sample-apiserver"
+  "k8s.io/metrics"
+)
+
 pushd "${KUBE_ROOT}" > /dev/null
+  # sanity check that staging directories do not exist in GOPATH
+  for R in "${STAGING_REPOS[@]}"; do
+    if [ -e "${GOPATH}/src/${R}" ]; then
```

Comment:
This will error one directory at a time if all 8 staging repos exist, meaning unless the user knows to delete all 8 the first time they hit this, they'll have to run the script 8 times to get all the dirs they need to remove. Perhaps we could collect all the dirs together and output the message once?